src/app/my-messages/my-messages.component.spec.ts

```diff
@@ -234,4 +277,56 @@ describe('MyMessagesComponent', () => {
       expect(matDialog.openDialogs[0].componentInstance).toEqual(jasmine.any(DeleteConfirmationDialogComponent));
     });
   }));
+
+  it('should display community information regarding to the notification type', fakeAsync(() => {
+    fixture.detectChanges();
+
+    const notificationCards = fixture.debugElement.queryAll(By.css(('.messages-card')));
+    let communityRow = getCommunityInformation(notificationCards[0]);
+
+    // INVITATION_TO_JOIN_COMMUNITY -> link to the community from a registration object
+    let actualData = communityRow.nativeElement.querySelector('a');
```

Comment:
I suggest to rename `actualData` to `linkToCommunity` to show what object we really work with.

Reply:
Thank you. Done.